pkg/controller/storagecluster/reconcile.go

```diff
@@ -931,6 +931,9 @@ func newExternalCephCluster(sc *ocsv1.StorageCluster, cephImage string) *cephv1.
 			External: cephv1.ExternalSpec{
 				Enable: true,
 			},
+			CrashCollector: cephv1.CrashCollectorSpec{
+				Disable: true,
+			},
 			DataDirHostPath: "/var/lib/rook",
```

Comment:
Actually, this is not needed. You can remove it.

Reply:
May I know why?
This is the field that rook is expecting right?.

Reply:
It's not needed anymore in the CephCluster external configuration. So please remove `DataDirHostPath`.

Reply:
My understanding is that `DataDirHostPath` is used for mons, and since we're not deploying any mons it's pointless to have it.

Reply:
Please check now.